EIPS/eip-X.md

```diff
@@ -0,0 +1,198 @@
+
+## Implementation
+
+The following implementations of eth2 Phase 0 exist and at the time of writing at least five are near production-ready and in testnet/audit phase:
+
+* [Cortex](https://github.com/NethermindEth/cortex) (.net)
+* [Lighthouse](https://github.com/NethermindEth/cortex) (rust)
+* [Lodestar](https://github.com/ChainSafe/lodestar) (javascript)
+* [Nimbus](https://github.com/status-im/nim-beacon-chain) (nim)
+* [Prysm](https://github.com/prysmaticlabs/prysm/) (go)
+* [Teku](https://github.com/pegasyseng/teku) (java)
+* [Trinity](https://github.com/ethereum/trinity/) (python)
+
+In addition to implementations of the spec, the deposit contract is [implemented in Solidity](), along with a [formal verification]() of the bytecode.
+
+## Security Considerations
+
+Eth2 is a major overhaul of the Ethereum's core consensus from PoW to a sharded PoS. There are inherent risks in this migration but there is a deep canon of research literature analyzing security and trade-offs. _The following only represents a high level selection of the resources available._
+
+* [Casper FFG](https://arxiv.org/abs/1710.09437)
+* [Combining GHOST and Casper](https://arxiv.org/abs/2003.03052)
+* [Sharding FAQ](https://eth.wiki/sharding/Sharding-FAQs)
+* [PoS FAQ](https://eth.wiki/en/concepts/proof-of-stake-faqs)
+* [Eth2 research compendium](https://notes.ethereum.org/@serenity/H1PGqDhpm?type=view)
+
+
+In addition to the research supporting this path, a number of audits and formal verification of specs, cryptography, and client implementations have been performed. _Many client and utility library audits are currently in progress and will be appended here upon completion._
+
+* [Eth2 Phase 0 Spec audit by Least Authority](https://leastauthority.com/blog/ethereum-2-0-specifications/)
+* [Gossipsub v1.1 audit by Least Authority](https://leastauthority.com/blog/audit-of-gossipsub-v1-1-protocol-design-implementation-for-protocol-labs/)
+* [Discv5 audit by Least Authority](https://leastauthority.com/blog/audit-of-gossipsub-v1-1-protocol-design-implementation-for-protocol-labs/)
+* [Formal Verification of Finality in eth2 by Runtime Verification](https://runtimeverification.com/blog/formally-verifying-finality-in-gasper-the-core-of-the-beacon-chain/)
+* [Formal Verification of Deposit Contract by Runtime Verification](https://github.com/runtimeverification/deposit-contract-verification/blob/master/deposit-contract-verification.pdf)
+* [Prysm client audit by Quantstamp](https://quantstamp.com/blog/ethereum-2-0-moves-closer-to-launch-with-quantstamp-audit-of-prysm)
+* [Lodestar utility libraries audit by Least Authority](https://leastauthority.com/blog/audit-of-chainsafe-utility-libraries/)
+
+And finally, the EF maintains eth2 spec, network, and client bug bounty programs
```

Comment:
```suggestion
Finally, the EF maintains the eth2 spec, network, and client bug bounty programs
```

Reply:
Can be parsed as "the EF maintains the eth2 network". Not the intended meaning i assume?